users/api_test.py

```diff
@@ -64,6 +75,20 @@ def test_fetch_users(prop, values, db_values):
     assert set(users) == set(found_users)
 
 
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "prop,values",
+    [["email", ["abc@example.com", "def@example.com", "ghi@example.com"]]],
+)
+def test_fetch_users_case_sens(prop, values):
```

Comment:
If we only have one set of values here does it make sense to inline the values instead?